components/applications-service/pkg/ingester/client.go

```diff
@@ -0,0 +1,14 @@
+package ingester
+
+type Client interface {
```

Comment:
💭 I kind of miss `context.Context` here. Is there some reason it doesn't make sense to be passed along / made use of? (I'm -- haha -- lacking context, too, so this might be no concern at all.)

Reply:
yup, we gotta fix this as well as in the storage layer! 😭